bibangamba/GitHubUsers
# #158851177 Add data binding to the project

#### What does this PR do?
Adds data binding to the project

#### Description of tasks to be done?
- add data binding library to app/build.gradle
- convert ui updating/rendering code to use data binding
- write tests for the new implementation

#### How can this be manually tested?
- clone the repo and check out the branch
- look through the UI implementation, there shouldn't be any findViewById calls and the project should be using `DataBindingUtils` to set the `contentView` and access layout widgets
- the application should also function exactly as before (run it on `develop` for comparison)

#### What are the relevant pivotal tracker stories?
[#158851177](https://www.pivotaltracker.com/story/show/158851177)
